drivers/flash/Kconfig.nor

```diff
@@ -27,11 +27,6 @@ config SPI_NOR_DRV_NAME
 	   - S25FL216K
 	   - MX25UM512
 
-config SPI_NOR_SPI_NAME
```

Comment:
Why this removal? (you use that option in your board's kconfig)

Reply:
@tbursztyka because all SPI related info should come from DTS (see #10894)

Reply:
Sure (see my comment about Kconfig.defconfig, it's using Kconfig option while it should get these from DTS), but that change should not be in this patch. It should come in a different one.

Reply:
Actually I would assume it is part of the first commit as it is introducing a new generic SPI driver

Reply:
True